tests/e2e/files/rulebooks/actions/test_run_playbook.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: Ruleset 1
+  hosts: all
+  sources:
+    - generic:
+        shutdown_after: 6
```

Comment:
Can you please include this value as envvar injected in the test? This would be a preparation for: https://issues.redhat.com/browse/AAP-9047